src/utils/EN.js

```diff
@@ -111,12 +111,12 @@ exports.ORDINAL_WORDS_PATTERN = '(?:'
     + ')';
 
 var TIME_UNIT = 
-    '(' + exports.INTEGER_WORDS_PATTERN + '|[0-9]+|an?(?:\\s*few)?|half(?:\\s*an?)?)\\s*' +
+    '(' + exports.INTEGER_WORDS_PATTERN + '|[0-9]+|[0-9]+\.[0-9]+|an?(?:\\s*few)?|half(?:\\s*an?)?)\\s*' +
     '(sec(?:onds?)?|min(?:ute)?s?|hours?|weeks?|days?|months?|years?)\\s*';
 
 var TIME_UNIT_STRICT = 
-    '([0-9]+|an?)\\s*' +
-    '(seconds?|minutes?|hours?|days?)\\s*';
+    '(?:[0-9]+|an?)\\s*' +
```

Comment:
I made these non-capturing groups so that both the strict and non-strict takes up the same number of sections within a match. Otherwise it'd need different indexes depending on strict mode or not and that seemed messier than needed.